github/search.go

```diff
@@ -172,18 +179,52 @@ func (c CodeResult) String() string {
 // GitHub API docs: https://developer.github.com/v3/search/#search-code
 func (s *SearchService) Code(ctx context.Context, query string, opt *SearchOptions) (*CodeSearchResult, *Response, error) {
 	result := new(CodeSearchResult)
-	resp, err := s.search(ctx, "code", query, opt, result)
+	resp, err := s.search(ctx, "code", &searchParameters{Query: query}, opt, result)
+	return result, resp, err
+}
+
+// LabelsSearchResult represents the result of a code search.
+type LabelsSearchResult struct {
+	Total             *int          `json:"total_count,omitempty"`
```

Comment:
s/Total/TotalCount/

Reply:
@gmlewis Please consider that this is consistent with the other search result structs. `RepositoriesSearchResult`, `RepositoriesSearchResult`, `CommitsSearchResult`, etc., all have `Total`, not `TotalCount`. We've had this discussion before and agreed it's better to keep it that way.

Reply:
OK, SGTM. Sorry I forgot about this exception to keeping the JSON tag consistent with the Go name.

Reply:
No problem! It's unfortunate we have to keep remembering these exceptions to the rules, but alas. Luckily, new types of search results aren't added all that often. :)